### PR TITLE
Small changes, typos, and a tiny bit more explanation.

### PR DESCRIPTION
**Pull request by jan-ferdinand**
• Add explanation that using Keras Functional API depends on
   understanding the Sequential API.
 • Fixed Markdown “Link Text” artifact.
 • Fixed typos: groud, congragulations, 10 images (actually 15).
 • Added plt.show() to suppress return value.
 • Changed “Accuracy” to “Loss” in plot for validation loss.
 • Increased verbosity level of .evaluate() to be consistent.
 • Split cell on result visualization to allow for faster generation
   of multiple results.

### CODE REVIEW

**Comment by buntyke** [comment]
Awesome! Thanks a lot! :smile:
